portal-api/src/constants/date-formats.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * For use with date-fns format method.
+ *
+ * e.g. 1st February 2024
+ *
+ * Equivalent to 'Do MMMM YYYY' in moment().format()
```

Comment:
Could you show an example of what it previews as?

Reply:
Do you mean the preview shown in VS-code?
> For use with date-fns format method.

>
>e.g. 1st February 2024

>
>Equivalent to 'Do MMMM YYYY' in moment().format()

>
>[https://date-fns.org/v3.3.1/docs/format](vscode-file://vscode-app/c:/Users/marrob/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-sandbox/workbench/workbench.html)